lib/opal/nodes/scope.rb

```diff
@@ -101,6 +101,24 @@ def def?
         @type == :def || @type == :defs
       end
 
+      def lambda?
+        iter? && @is_lambda
+      end
+
+      def is_lambda!
```

Comment:
The lint message can be fixed by append `# rubocop:disable Naming/PredicateName` to the `def` line